packages/data-stores/src/site/test/reducer.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * These tests shouldn't require the jsdom environment,
+ * but we're waiting for this PR to merge:
+ * https://github.com/WordPress/gutenberg/pull/20486
```

Comment:
This has been merged, and I think it's been published as [`@wordpress/priority-queue`](https://www.npmjs.com/package/@wordpress/priority-queue) 1.5.1. Since our `package-lock` is apparently on `^1.5.0`, we might be able to drop the `jsdom` env?

Reply:
This one's still failing for me after running a fresh `npm ci`, so let's keep it for now.

Reply:
Same, seems like we need to hit it with a bigger update-dependencies-hammer